### PR TITLE
adjust font size

### PR DESCRIPTION
**Pull request by spandanabaruah**

### CODE REVIEW

File: components/d2l-activity-editor/d2l-activity-assignment-editor/d2l-activity-assignment-editor-submission-and-completion.js [hunk 1]
```diff
@@ -93,9 +93,9 @@ class ActivityAssignmentSubmissionAndCompletionEditor extends ActivityEditorMixi
 	_renderAssignmentType() {
 		return html `
 			<div id="assignment-type-container">
-				<h3 class="assignment-type-heading d2l-heading-4">
```
Comment (wongvincent) [inline components/d2l-activity-editor/d2l-activity-assignment-editor/d2l-activity-assignment-editor-submission-and-completion.js:96]: can remove the CSS on line 36

File: components/d2l-activity-editor/d2l-activity-assignment-editor/d2l-activity-assignment-type-editor.js [hunk 2]
```diff
@@ -129,11 +129,11 @@ class AssignmentTypeEditor extends ActivityEditorMixin(LocalizeMixin(MobxLitElem
 
 		return html`
 			<div ?hidden=${!isReadOnly} id="read-only-assignment-type-container">
-				<div class="d2l-body-standard">${folderTypeText}</div>
```
Comment (wongvincent) [inline components/d2l-activity-editor/d2l-activity-assignment-editor/d2l-activity-assignment-type-editor.js:132]: can remove references to bodyStandard on line 1 and line 16

File: components/d2l-activity-editor/d2l-activity-assignment-editor/d2l-activity-assignment-editor-submission-and-completion.js [hunk 3]
```diff
@@ -93,9 +89,9 @@ class ActivityAssignmentSubmissionAndCompletionEditor extends ActivityEditorMixi
...
```
Comment (wongvincent) [inline components/d2l-activity-editor/d2l-activity-assignment-editor/d2l-activity-assignment-editor-submission-and-completion.js:96]: oops, sorry also missed that "assignment-type-heading" CSS on line 40 can be removed.

Comment (spandanabaruah) [reply components/d2l-activity-editor/d2l-activity-assignment-editor/d2l-activity-assignment-editor-submission-and-completion.js:96]: ah I should've seen that! pushing a new commit with this change

**Pull request merged**